compiler/damlc/tests/daml-test-files/PreludeTest.daml

```diff
@@ -387,3 +389,18 @@ testDateOverflow = scenario do
 
 testDateUnderflow = scenario do
     pure $ date 2000 Jan 0
+
+testDateRange = scenario do
+    assert (length [(date 2019 Jan 1)..(date 2020 Dec 31)] == 731)
+
+testDateMinBound : Scenario Date = do
+    return minBound
+
+testDateMaxBound : Scenario Date =  do
+    return maxBound
```

Comment:
These two tests don't test anything. I think what we want instead is something like
```suggestion
testDateMinBound : Scenario Date = do
    fromEnum minBound === 0

testDateMaxBound : Scenario Date =  do
    fromEnum maxBound === <whatever the right number is>
```